### ISSUE TITLE
Char constructor should use fromCharCode

### ISSUE THREAD

**Issue by lijunle** [fsprojects/Fable#106]
### Description

The code `char 97` is convert to `(97).toString()`, should be `String.fromCharCode(97)`

### Expected behavior

`char 97 = 'a'`

### Actual behavior

`char 97 -> (97).toString() -> "97"`

### Known workarounds

I don't know any.

### Related information 

* Operating system: Windows 10
* Branch: Install from NPM - version 0.2.2
* .NET Runtime 4.6.1
* F# version 4.4

**Comment by lijunle** [comment]
Another compilation issue:

```f#
let charListToString chars =
    System.String(List.toArray chars)
```

Convert to

```js
var charListToString = exports.charListToString = function (chars) {
  return _fableCore.String.fsFormat(Array.from(chars));
};
```

I am using `System.String`, it is not related to `String.fsFormat`. :disappointed:

**Comment by fdcastel** [comment]
Thank you for the report!

Fixed in 35f6b4b5509da0626397d95421461cb7090469af.

Please take a look at it and let me know if this solves the problem for you.

**Comment by lijunle** [comment]
Thanks @fdcastel for fixing the string constructor, is that your changes fixing the char constructor too?

**Comment by alfonsogarciacaro** [comment]
Yeah, the changes made by @fdcastel have also fixed various String constructors as you can see in the new [unit tests](http://wiki.ecmascript.org/doku.php?id=strawman:do_expressions) :) I have published the new versions (fable-compiler 0.2.3 & fable-core 0.0.12). Please give them a try and feel free to close the issue if it's working.

**Comment by lijunle** [comment]
OK, I will try that later. Thanks very much!
